model/project.go

```diff
@@ -415,6 +418,12 @@ type BuildVariant struct {
 	DisplayTasks []patch.DisplayTask    `yaml:"display_tasks,omitempty" bson:"display_tasks,omitempty"`
 }
 
+// CheckRun is used to provide information about a github check run.
+type CheckRun struct {
+	// PathToOutputs is a local file path to an output json file for the checkrun
+	PathToOutputs string `yaml:"path_to_outputs" bson:"path_to_outputs"`
```

Comment:
If this struct only holds one string now, does it need to be a struct anymore? Or are we going to be adding new fields in the future potentially

Reply:
yes, I added a note up top. 

> While it would seem that it makes more sense for create_check_run to be a string instead of a struct, I made it a struct so that we can later accommodate [EVG-21103](https://jira.mongodb.org/browse/EVG-21103) without requiring users to change their config files.